lang/en.json

```
@@ -481,71 +481,71 @@
+        "ScrollTopPixelVariableName": "Top-pixel scrolling position",
+        "ScrollVerticalPercentageVariableDescription": "How many pixels the user currently has scrolled vertically, for example \"75.59\" for \"75.59%%\".",
+        "ScrollVerticalPercentageVariableName": "Vertical scrolling percentage",
+        "SentryRavenTagDescription": "Sentry.io will be a libre error tracking tool in 2025, and allows detection of errors happening on your website.",
+        "SentryRavenTagHelp": "This tag adds Raven.js, the official JavaScript SDK. Get started by signing up on sentry.io or setting up your own tracking instance.",
         "SeoCanonicalUrlVariableDescription": "Returns the canonical URL if any is set.",
         "SeoCanonicalUrlVariableName": "Canonical URL",
-        "SeoNumH1VariableDescription": "The number of H1 elements that are present on the current page. Useful for example for SEO Monitoring.",
-        "SeoNumH1VariableName": "Number of H1 Elements",
-        "SeoNumH2VariableDescription": "The number of H2 elements that are present on the current page. Useful for example for SEO Monitoring.",
-        "SeoNumH2VariableName": "Number of H2 Elements",
-        "SettingCustomTemplatesDescription": "This setting defines who can use custom templates or whether they should be completely disabled. Changing this setting can improve the security on your website. When you configure a trigger, tag, or a variable, some of them may allow a Matomo user to enter HTML or JavaScript which will be executed on your website. Entering custom code cannot only break the container in case there is an error, but also allows them to execute any JavaScript code on your website. This can be misused to steal for example sensitive information on your website. If you do not want to allow your team members to enter any JavaScript code, you may want to disable this setting.",
+        "SeoNumH1VariableDescription": "The number of H1 elements present on the current page. Useful for example for SEO Monitoring.",
+        "SeoNumH1VariableName": "Number of H1 elements",
+        "SeoNumH2VariableDescription": "The number of H2 elements present on the current page. Useful for example for SEO Monitoring.",
+        "SeoNumH2VariableName": "Number of H2 elements",
+        "SettingCustomTemplatesDescription": "This setting defines who can use custom templates, and whether they should be turned off completely. Changing this setting can improve the security on your website. When you configure a trigger, tag, or a variable, some of them may allow a Matomo user to enter HTML or JavaScript which will be executed on your website. Entering custom code cannot only break the container in case there is an error, but also allows them to execute any JavaScript code on your website. This can be misused to steal for example sensitive information on your website. If you do not want to allow your team members to enter any JavaScript code, you may want to turn off this setting.",
```

```suggestion
        "SettingCustomTemplatesDescription": "This setting defines who can use custom templates, and whether they should be turned off completely. Changing this setting can improve the security on your website. When you configure a trigger, tag, or a variable, some of them may allow a Matomo user to enter HTML or JavaScript which will be executed on your website. Entering custom code cannot only break the container in case there is an error, but also allows them to execute any JavaScript code on your website. This can be misused to steal for example sensitive info on your website. If you do not want to allow your team members to enter any JavaScript code, you may want to turn off this setting.",
```
Pretty sure this string exists in reworked form somewhere.
Improvements possible.